bitnami/postgresql/README.md

```diff
@@ -647,7 +647,33 @@ This label will be displayed in the output of a successful install.
 - The Bitnami PostgreSQL image is non-root by default. This requires that you run the pod with `securityContext` and updates the permissions of the volume with an `initContainer`. A key benefit of this configuration is that the pod follows security best practices and is prepared to run on Kubernetes distributions with hard security constraints like OpenShift.
 - For OpenShift, one may either define the runAsUser and fsGroup accordingly, or try this more dynamic option: volumePermissions.securityContext.runAsUser="auto",securityContext.enabled=false,containerSecurityContext.enabled=false,shmVolume.chmod.enabled=false
```

Comment:
Since there is already a paragraph related to this topic, what do you think about editing it using the updated values instead of creating a new whole section?

Reply:
I meant to delete this list item -- I apologize for my error.

It is not technically possible to put a code-fenced extent within a list item, and since I offer an example file as well as an example command, it would lead to awkward formatting, indeed.

Reply:
You should be able to use a single line command as follows:
```
helm install ... --set primary.podSecurityContext.enabled: false,primary.podSecurityContext.fsGroup: "auto",primary.containerSecurityContext.enabled: false,primary.containerSecurityContext.runAsUser: "auto",volumePermissions.enabled: false,volumePermissions.containerSecurityContext.runAsUser: "auto"
```

The different parameters were made using the code block you shared